packages/PlanLimitsUI/src/api/index.ts

```diff
@@ -19,8 +19,7 @@ const determineBackendUri = (hostname: string) => {
     return 'https://data.gw-eop-dev.tech';
   }
 
-  return 'https://data.gw-eop-dev.tech';
-  // return 'http://localhost:8080';
+  return 'http://localhost:8080';
```

Comment:
Whoops, this was a hangover from previous work (I must have been pointing the UI at the dev backend)